src/Simplex/Chat.hs

```diff
@@ -3109,6 +3153,7 @@ chatCommandP =
       "/_call end @" *> (APIEndCall <$> A.decimal),
       "/_call status @" *> (APICallStatus <$> A.decimal <* A.space <*> strP),
       "/_call get" $> APIGetCallInvitations,
+      "/_profile @" *> (APIUpdateOtherProfile <$> A.decimal <* A.space <*> ("userPreferences=" *> jsonP)),
```

Comment:
the name is misleading as you cannot really change profile per contact, you are only changing the preference.

The command in rfc is `/_set prefs @ <json>`